packages/slice-machine/src/legacy/lib/models/common/widgets/Link/Form.jsx

```diff
@@ -6,12 +6,11 @@ import { createFieldNameFromKey } from "@/legacy/lib/forms";
 import { DefaultFields } from "@/legacy/lib/forms/defaults";
 import { CheckBox } from "@/legacy/lib/forms/fields";
 
-import { DisplayTextCheckbox } from "./components";
+import { DisplayTextCheckbox, RepeatableCheckbox } from "./components";
 
 // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
 const FormFields = {
   ...DefaultFields,
-  allowTargetBlank: CheckBox("Allow target blank", false, true),
```

Comment:
@levimykel Correct me if I'm wrong, but this is not needed right? Since we're not mapping over `FormFields` to render `allowTargetBlank`, we can just set the default [here](https://github.com/prismicio/slice-machine/blob/ccffe983f91b1b66cddb12952bf1b372bbd64fe6/packages/slice-machine/src/legacy/lib/models/common/widgets/Link/index.ts#L109) and remove this.

Reply:
@jomifepe I don't remember there being a specific need for this to be there. As long as everything is working as expected, then it seems that this isn't needed.